src/runtime.js

```diff
@@ -756,6 +756,7 @@ class MultidocManager {
                 shadowRoot,
                 dev().assertElement(doc.body),
                 /* deep */ false);
+            //NOTE(torch2424), done in importshadowbody
```

Comment:
A useful comment? Let's remove in one place or the other. I think it should go here, b/c `amp-shadow` concept only belongs to the top-level PWAMP org, not to any generic shadow root.